l10n_pt_vat/README.rst

```diff
@@ -0,0 +1,34 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Portugal - VAT
```

Comment:
Next time prefer to have a separate PR for each model: smaller PRs are easier to review, and longer PRs take much longer to get reviewed.